### PR TITLE
Attempting to make headings all conform to the same style

### PR DESCRIPTION
**Pull request by TravisTheTechie**
Re: https://github.com/less/less-docs/issues/346

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,4 +70,3 @@ body { padding-top: 80px; font-size: 12px }
 
 * If you would like to request a feature, suggest an improvement, or report a bug, please [submit an Issue]({{ site.codeissues }}).
 * Feature requests are more likely to get attention if you include a clearly-described use case.
-* If you wish to submit a pull request, please [read this first]({{ site.codebasemaster }}CONTRIBUTING).md.
```
Comment (TravisTheTechie) [inline CONTRIBUTING.md]: {{ }} macros don't work + plus you're just linking to the same doc.

File: content/usage/command-line-usage.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ title: Command Line Usage
 
 <span class="warning">Heads up! If the command line isn't your thing, learn more about [GUIs for Less](#guis-for-less).</span>
 
-### Installing lessc for use globally
+### Installing `lessc` for Use Globally
```
Comment (TravisTheTechie) [inline content/usage/command-line-usage.md]: Right way to refer to `lessc`?

File: content/usage/using-less-in-the-browser.md [hunk 3]
```diff
@@ -206,7 +206,7 @@ Default: `1000`
 
 The amount of time (in milliseconds) between polls while in watch mode.
 
-### relativeUrls
+#### relativeUrls
```
Comment (TravisTheTechie) [inline content/usage/using-less-in-the-browser.md]: Double check this?

File: CONTRIBUTING.md (hunk 1 above)
Comment (lukeapage) [inline CONTRIBUTING.md]: It was probably copied from somewhere else. This is fine..

File: content/usage/command-line-usage.md (hunk 2 above)
Comment (lukeapage) [inline content/usage/command-line-usage.md]: If it looks okay on the site (isn't styled weirdly), then yes

File: content/usage/using-less-in-the-browser.md (hunk 3 above)
Comment (lukeapage) [inline content/usage/using-less-in-the-browser.md]: that's correct, thanks

File: content/usage/command-line-usage.md (hunk 2 above)
Comment (lukeapage) [inline content/usage/command-line-usage.md]: We use it in other places so yes this is fine

**Pull request merged**